### PR TITLE
add pytorch 22.11 to test matrix

### LINKED ISSUE #5912: Add PyTorch docker CI tests for the LTS versions of MONAI
**Is your feature request related to a problem? Please describe.**
As we are using MONAI 1.0.1 in the MONAI toolkit, would be nice to add this version to the PyTorch docker CI tests.
And for the future MONAI versions, we should add the LTS versions (used in MONAI toolkit) to the CI tests.

### PR DESCRIPTION
**Pull request by YanxuanLiu**
Signed-off-by: YanxuanLiu <yanxuanl@nvidia.com>

Fixes #5912  .

### Description

1. removed 21.02 and 21.10 from cron
2. add 22.09 and 22.11 to blossom jobs

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: .github/workflows/cron.yml [hunk 1]
```diff
@@ -119,7 +119,7 @@ jobs:
     if: github.repository == 'Project-MONAI/MONAI'
     strategy:
       matrix:
-        container: ["pytorch:21.02", "pytorch:21.10", "pytorch:22.12"]  # 21.02, 21.10 for backward comp.
+        container: ["pytorch:22.12"]
```
Comment (Nic-Ma) [inline .github/workflows/cron.yml:122]: May I know why remove them?

Thanks.

Comment (wyli) [reply .github/workflows/cron.yml:122]: Those are deprecating tests for early versions of clara train 4.0/4.1, I don't think we should still spend resources testing...

Comment (wyli) [reply .github/workflows/cron.yml:122]: Please also add 22.11 here, this is will be part of the nightly tests. @YanxuanLiu

Comment (Nic-Ma) [reply .github/workflows/cron.yml:122]: OK, I see.

Thanks.

**Review by wyli — APPROVED** [review]

**Pull request merged**